paths/questions.yaml

```diff
@@ -22,6 +22,14 @@ post:
       description: Slug Representation of the feedback in the URL. Requirement depends on api-configuration
       required: false
       type: string
+    - name: answer_id
+      description: uuid of answer which is being forked
+      in: query
```

Comment:
I think that should be body. I know that is not consistent in the whole of swagger, but it also creates confusion every time.

Reply:
agree, should I change it for this file alone?

Reply:
you could also go though all files, but that would be outside the scope of this ticket. I think only this file will be ok